### ISSUE TITLE
🐛 Bug Report: node-appwrite(v11.0.0) sdk called os.type, but this function did not exist.

### ISSUE THREAD

**Issue by usamashafiqs** [appwrite/appwrite#6216]
### 👟 Reproduction steps

![Screenshot 2023-09-11 184758](https://github.com/appwrite/appwrite/assets/18592683/cb1861d0-6d90-43e4-b7b0-4ff129252b89)

### 👍 Expected behavior

It should connect to the AppWrite backend.

### 👎 Actual Behavior

It is actually not connecting to the backend.

### 🎲 Appwrite version

Version 1.4.x

### 💻 Operating system

Windows

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@usamashafiqs, thanks for raising this issue! 🙏🏼 Would you please provide more context about where this code is running?

**Comment by usamashafiqs** [comment]
@stnguyen90, This code is running in the browser.

**Comment by stnguyen90** [comment]
@usamashafiqs, would you please try using version [13.0.0 of the appwrite package](https://www.npmjs.com/package/appwrite) instead of the node-appwrite package?

**Comment by usamashafiqs** [comment]
@stnguyen90 I was connecting with an API key; Standerd Appwrite package didn't have an API key option.

**Comment by stnguyen90** [comment]
@usamashafiqs, no, it is by design. You should not be using an API key client-side because that leaks your key to the world. Nothing client side can be kept secret.

**Comment by usamashafiqs** [comment]
@stnguyen90  I am using Traefik [ForwardAuth](https://doc.traefik.io/traefik/middlewares/http/forwardauth/) to protect the app; it is an internal tool.
